_posts/en/newsletters/2021-11-24-newsletter.md

```diff
@@ -0,0 +1,82 @@
+---
+title: 'Bitcoin Optech Newsletter #176'
+permalink: /en/newsletters/2021/11/24/
+name: 2021-11-24-newsletter
+slug: 2021-11-24-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter links to a discussion about how to allow LN users
+to choose between higher fees and higher payment reliability and
```

Comment:
That doesn't quite work, because the "includes" next applies to both "our regular...", "announcements..." and "summaries...".

Perhaps just split into a new sentence:

"...choose between higher fees and higher payment reliability. Also included are ..."